packages/react-scripts/template/README.md

```diff
@@ -207,7 +207,7 @@ To configure the syntax highlighting in your favorite text editor, head to the [
 
 ## Displaying Lint Output in the Editor
 
->Note: this feature is available with `react-scripts@0.2.0` and higher.  
```

Comment:
Whitespace is significant in markdown. Please leave those two spaces in, they are causing the line to break. They were intentional.